src/elm/Util/TravelCard.elm

```diff
@@ -24,3 +26,55 @@ formatAnonymized =
 formatSignificant : String -> String
 formatSignificant =
     NF.formatString [ NF.Digits 2, NF.Space, NF.Digits 7 ]
+
+
+serverErrorToString : (String -> Result error String) -> Error -> String
+serverErrorToString encode error =
+    case error of
+        BadStatus { status, body } ->
+            case status.code of
+                500 ->
+                    "Det skjedde en feil med tjenesten. Prøv igjen senere."
+
+                409 ->
+                    "Dette t:kortet eksisterer ikke eller er allerede registrert."
+
+                400 ->
+                    case encode body of
+                        Ok errorMessage ->
+                            translateErrorString errorMessage
+
+                        _ ->
+                            "Innsendt informasjon ser ut til å ikke stemme. Prøv igjen er du snill."
+
+                _ ->
+                    "Unknown error"
+
+        _ ->
+            "Fikk ikke kontakt med tjenesten. Sjekk om du er på nett og prøv igjen."
+
+
+
+-- Invalid travel card id length
```

Comment:
Dette kan vel fjernes?

Reply:
Eventuelt legges inn som kommentarer over hver string i funksjonen under, typ:

```elm
if check "id length" then
    -- Invalid travel card id length
    "t:kortnummeret må bestå av 16 siffer"
...
```

Reply:
Jeg var alt for kjapp i å prøve å få igjennom her. Skal bort selvsagt